.github/workflows/phpstan.yml

```
@@ -0,0 +1,22 @@
+ 
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.3'
+          coverage: none
+ 
+      - name: Install composer dependencies
+        run: composer install -n --prefer-dist
+ 
+      - name: Run Static Analysis
+        run: ./vendor/bin/phpstan --error-format=github
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)